lib/mailroom/imap.ex

```diff
@@ -160,6 +168,10 @@ defmodule Mailroom.IMAP do
     end
   end
 
+  def uid_search(pid, query) do
```

Comment:
Please add a test for this. If you start an IMAP session with a server you control with `debug: true` you can get the server output and then, removing sensitive information, create a test like the others in imap_test.exs

Reply:
done